howto/how-to-set-up-a-blaster.md

```diff
@@ -0,0 +1,228 @@
+| `font` | Name of the preset. |
+| `full` | Sound made when the weapon is full of ammo. |
+| `hum` | Constant sound looping while not firing. |
+| `jam` | Sound made when the weapon jam.s |
+| `unjam` | Sound made when unjamming the blaster. |
+| `mode` | Sound made when switching mode (OS 6+: when `mdkill`, `mdstun` and/or `mdauto` not present) |
+| `plioff` | Played while retracting the PLI bargraph. | 
+| `plion` | Played while extending the PLI bargraph. |
+| `poweron` | If this file is present, the blaster will start once it is turned on. If not, *you will need a Power button to power it on*. |
+| `range` |  Sounds of increasing weapon range/power. Currently not implemented. |
+| `stun` | Firing sound for stun mode. |
+| `reload` | Reloading sound. |
+| `mdkill` | (OS 6+) Sound made when switching to KILL mode |
+| `mdstun` | (OS 6+) Sound made when switching to STUN mode |
+| `mdauto` | (OS 6+) Sound made when switching to AUTO mode |
+
+*If no `mdkill`, `mdstun`, `mdauto` nor `mode` are present Talkie voice speaks selected mode.*
+
+# OLED Animations
+[TODO]
+
+# Configuration
+Then comes the time to configure the board's `config.h`. Proffieboard uses the same hardware and OS for blasters, thermal detonators and blasters. Thus, we will assume you are somewhat familiar with the saber configuration. If you find defines or elements that are not documented here, they are probably documented on the saber side. We will provide links to the relevant sections of the define for a more complete explanation.
+So, we will propose a sample config for the hardware that we proposed as an example, and then we will go over the particulars of the blaster define.
+
+## Configuration Sample
+
+```
+/****** CONFIG_TOP blaster defines  ****/
+#ifdef CONFIG_TOP
+
+#include "proffieboard_v2_config.h"
+#define NUM_BLADES 2    // We use a WS2812B LED string for the barrel, plus a straight red LED for an accent.
+#define NUM_BUTTONS 2   // First button is fire, second adds Mode Select, you can add Power, Reload or Clip Detect.
+                        // As always, you have to add them in `CONFIG_BUTTONS`.
+const unsigned int maxLedsPerStrip = 144;
+#define CLASH_THRESHOLD_G 2.0         //Affects the shaking for unjamming your weapon.
+#define VOLUME 1500                   // 0-3000 usual range. Don't overload your speaker.
+
+// Stock required defines
+#define ENABLE_AUDIO
+#define ENABLE_MOTION
+#define ENABLE_WS2811
+#define ENABLE_SD
+
+//OLED Specific
+//#define ENABLE_SSD1306		            //To enable stock 128x32 monochrome SSD1306 OLED
+//#define INCLUDE_SSD1306               //To enable a different aspect ratio, needs some further configuration.
+
+//Blaster specific.
+#define ENABLE_BLASTER_AUTO          // If you desire to enable AUTO mode.
+#define BLASTER_SHOTS_UNTIL_EMPTY 15 // The capacity of your weapons rounds.
+#define BLASTER_JAM_PERCENTAGE 1     //  0-100. If not defined, random.
+
+
+#endif
+
+/****** CONFIG_PROP blaster defines  ****/
```

Comment:
This is redundant.
The #ifdef CONFIG_PROP already declares this to be the CONFIG_PROP section.